go-controller/pkg/clustermanager/node/node_allocator.go

```diff
@@ -149,6 +150,7 @@ func (na *NodeAllocator) releaseHybridOverlayNodeSubnet(nodeName string) {
 // HandleAddUpdateNodeEvent handles the add or update node event
 func (na *NodeAllocator) HandleAddUpdateNodeEvent(node *corev1.Node) error {
 	defer na.recordSubnetCount()
+	na.clearInitialNodeNetworkUnavailableCondition(node)
```

Comment:
@JacobTanenbaum I'd actually keep the node allocator stuff focused on subnets, and just put this into `func (h *networkClusterControllerEventHandler) AddResource(obj interface{}, fromRetryLoop bool) error {` right after `HandleAddUpdateNodeEvent()` runs; and put `clearInitialNodeNetworkUnavailableCondition()` into network_cluster_controller.go too

Reply:
done